src/Tasks/Microsoft.NET.Build.Tasks/ResolvePackageAssets.cs

```diff
@@ -45,15 +45,10 @@ public sealed class ResolvePackageAssets : TaskBase
         public string ProjectPath { get; set; }
 
         /// <summary>
-        /// TFM to use for compile-time assets.
+        /// TargetFramework to use for compile-time assets.
         /// </summary>
         [Required]
-        public string TargetFrameworkMoniker { get; set; }
-
-        /// <summary>
-        /// TPM to use for compile-time assets. 
-        /// </summary>
-        public string TargetPlatformMoniker { get; set; }
+        public string TargetFramework { get; set; }
```

Comment:
this should have platform or not? net5.0 or net5.0-windows

Reply:
and reminds why we use that instead of targetframeworkmoniker + targetplatformmoniker

Reply:
It should be the raw `TargetFramework` value from the project.  So generally `net5.0-windows`.  It's the "alias".

Reply:
We want to key our assets file lookups off of the (potentially aliased) `TargetFramework`.  So if someone uses custom logic to multi-target between `net5.0-x86` and `net5.0-x64`, there will be different assets file sections for each of those even though they both have the same TFM and TPM.  (Note that I don't think this works end-to-end yet, but this future-proofs our logic).